### ISSUE TITLE
Doc Error on Box Widget

### ISSUE THREAD

**Issue by pkeogan** [ColorlibHQ/AdminLTE#2072]
### Issue type:
- [ ] Feature request 
- [ ] Bug report       
- [x] Documentation   

### Environment:
* AdminLTE Version:  2.4.8
* Operating System:  n/a
* Browser (Version):  n/a

### Description:
Doc Error: https://adminlte.io/docs/2.4/js-box-widget

The doc is stating that the box widget default is the follow:
```
$('#my-box').boxWidget({
  animationSpeed: 500,
  collapseTrigger: '#my-collapse-button-trigger',
  removeTrigger: '#my-remove-button-trigger',
  collapseIcon: 'fa-minus',
  expandIcon: 'fa-plus',
  removeIcon: 'fa-times'
});
```

When the actually code is the following: (pulled from https://github.com/almasaeed2010/AdminLTE/blob/master/build/js/BoxWidget.js)
```
$('.box').boxWidget({
    animationSpeed : 500,
    collapseTrigger: '[data-widget="collapse"]',
    removeTrigger  : '[data-widget="remove"]',
    collapseIcon   : 'fa-minus',
    expandIcon     : 'fa-plus',
    removeIcon     : 'fa-times'
});
```

This doc error might be causing the problems newer users to adminLTE are having when trying to reinitialize box widgets after ajax loads or dynamic creation.

**Comment by marcelogarbin** [comment]
I downloaded the AdminLTE here, and I executed, by clicking on the "minus" box, it generates the collapse effect. But this error occurs. Would you know what that might be?

![image](https://user-images.githubusercontent.com/9335588/55409348-4410ec00-5538-11e9-8625-0aa4bf2f9145.png)

![erro-adm](https://user-images.githubusercontent.com/9335588/55409772-06609300-5539-11e9-8575-fef1c794f4ac.gif)

**Comment by REJack** [comment]
Is this Issue still open with the latest Version (v2.4.10)?

**Comment by pkeogan** [comment]
not sure - docs arent on github so i cant make a pull

**Comment by REJack** [comment]
This issue is fixed with v2.4.11
